assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -143,12 +144,12 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
  *       in the frontend.
  */
 const CartEditor = ( { className, attributes, setAttributes } ) => {
-	if ( attributes.isPreview ) {
-		return cartBlockPreview;
```

Comment:
It looks like `cartBlockPreview` is not used anywhere else, right? Can we remove `/assets/js/previews/cart-block.js` and `/assets/img/cart-preview.svg`? Same for Checkout preview.